.ask/config

```diff
@@ -2,18 +2,7 @@
   "deploy_settings": {
     "default": {
       "skill_id": "",
-      "was_cloned": false,
-      "merge": {
-        "skillManifest": {
-          "apis": {
```

Comment:
@Kazushige-Yoshida-A  @franklin-lobb will removing this from the config file prevent the lambda function from being created?

Reply:
I confirmed that this is an issue. 

Attempting to run `ask deploy` without this information in the .ask/config file causes the cli to fail uploading the content.

I made the change locally and will merge.